File: RazorClassLibrary1/ChartJS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.JSInterop;
using System.Threading.Tasks;

namespace RazorClassLibrary1
{
    public static class ChartJS
    {
        public static ValueTask CreateChart(IJSRuntime jsRuntime, string canvasId, string chartType, object data, object options)
        {
            return jsRuntime.InvokeVoidAsync("chartInterop.createChart", canvasId, chartType, data, options);
        }
    }
}

File: RazorClassLibrary1/wwwroot/chartJS.js
window.chartInterop = {
    createChart: (canvasId, chartType, data, options) => {
        const ctx = document.getElementById(canvasId).getContext('2d');
        new Chart(ctx, {
            type: chartType,
            data: data,
            options: options
        });
    }
};
